mewarpx/mewarpx/assemblies.py

```diff
@@ -294,3 +299,184 @@ def _install_in_simulation(self):
             implicit_function=self.implicit_function,
             potential=self.V
         )
+
+
+class Rectangle(Assembly):
+    """An infinitely long rectangular prism pointing in the y-direction"""
```

Comment:
```suggestion

    """An infinitely long rectangular prism pointing in the y-direction"""
```

Reply:
Please expand this docstring with information about what the regions 0-5 mean and how the normal calculation progresses - OR move the normal calculation above the other functions in support of it, and add this comment there. Either way, I'd like to have the info going into each function as I'm reading it to understand what that function is doing. Or add a docstring to each function with more info. Any of those could work, but right now the normal calculation is non-obvious.